### ISSUE TITLE
Dead UI Link ETH about section

### ISSUE THREAD

**Issue by wesleygravess** [shapeshift/web#2167]
### Overview

In the about section on the ETH account and asset page, there's a dead link pop up that appears when you hover over the underlined text, "smart contract platform."

### References and additional details

1. go to app.shapeshift
2. log in with your wallet of choice
3. go to the ETH accounts or asset page.
4. go to the about section
5. hover over the underlined text, "smart contract platform"
6. noticed the dead link pop up that appears

### Acceptance Criteria

Either the hyperlink should be added to the "smart contract platform" text in the about section or the dead link pop up should be removed. Not sure what the expected behavior should be here

### Screenshots/Mockups

<img width="330" alt="Screenshot 2022-07-14 124250" src="https://user-images.githubusercontent.com/99363597/179045254-4d724b2a-7e82-4a1e-bad9-19cd2312af3e.png">

**Comment by gomesalexandre** [comment]
@shapeshift/product Please have a look at the screenshots in the PR - This will do for now as far as fixing links goes, but we might want to iterate on this with our own in-house, improved description, as it's a bit outdated.

**Comment by DiggyDiggy2** [comment]
yeah I think we can remove the link. I'll have @decentralized-j take a pass at updating the copy there.
